tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -589,7 +589,8 @@ SpirvInstruction *DeclResultIdMapper::getDeclEvalInfo(const ValueDecl *decl,
 }
 
 SpirvFunctionParameter *
-DeclResultIdMapper::createFnParam(const ParmVarDecl *param) {
+DeclResultIdMapper::createFnParam(const ParmVarDecl *param,
+                                  uint32_t param_dbg_idx) {
```

Comment:
To stay consistent with variable naming conventions in spiregg, please use `paramDebugIndex` instead of `param_dbg_idx`

Reply:
done